usr/lib/sticky/sticky.py

```diff
@@ -482,7 +482,8 @@ def apply_format(self, m, format_type):
 
     def remove(self, *args):
         # this is ugly but I'm not sure how to make it look better :)
-        if (self.app.settings.get_boolean('disable-delete-confirm') or
+        if ((not self.title.get_text() and not self.buffer.get_internal_markup()) or
```

Comment:
Could you please move this new condition to after the settings check? It probably wont make a noticeable difference either way in the vast majority of situations, but I can see it potentially causing a noticeable difference in the case where the user is trying to delete a particularly large note on a slow system with the delete confirmation setting disabled.

Reply:
No problem, that's a good point.